quickstart.sh

```diff
@@ -63,6 +63,15 @@ fi
 echo "Mounting ${PROJECT_DIR} to container as Project Directory /runner/project"
 echo "Creating Container ${CONTAINER_NAME} from image ${IMAGE_FULL_NAME}"
 
+echo "Checking if ssh-agent is running..."
+if pgrep -x "ssh-agent" >/dev/null
+then
+    echo "ssh-agent OK"
+else
+    echo "ssh-agent is stopped, restarting"
+    eval `ssh-agent -s` 
```

Comment:
Rather than restarting, we should exit to notify the user that the agent isn't running, like what we do above with Docker:

`echo "Checking if Docker is running..."
{ docker info >/dev/null 2>&1; echo "Docker OK"; } || { echo "Docker is required and does not seem to be running - please start Docker and retry" ; exit 1; }`

Reply:
Right. I have made this change so it will now request the user to start the ssh-agent. I'm not convinced that this is the best approach, as it has to be started in a specific way otherwise the end user will get stuck trying to figure out SSH_AUTH_SOCK issues.